src/networks/DFP.ts

```diff
@@ -19,7 +19,7 @@ class DfpAd implements INetworkInstance {
 
     const {
       el: { id },
-      configuration: { sizes, targeting, path, breakpoints },
+      configuration: { sizes, targeting, path, breakpoints, fluid },
```

Comment:
I thought we not longer need this since we can pass it in an array??

Reply:
oh! Yep, you're right, missed that. Sec.